.github/workflows/check-pl-compat.yaml

```diff
@@ -102,6 +102,20 @@ jobs:
         LIGHTNING_GIT_TAG_VALUE=latest_release \
         make runtime
 
+    - name: Install PennyLane-Lightning (latest)
+      if: ${{ inputs.lightning == 'latest' }}
+      uses: actions/checkout@v3
+      with:
+        repository: PennyLaneAI/pennylane-lightning
+        path: pennylane-lightning
+    - if: ${{ inputs.lightning == 'latest' }}
+      run: |
+        pip install --upgrade ./pennylane-lightning
+
+    - name: Install PennyLane-Lightning (stable)
+      if: ${{ inputs.lightning == 'stable' }}
+      run: |
+        pip install --upgrade pennylane-lightning
```

Comment:
Thanks for catching this!

Reply:
Do you think it's issue that we install lightning before PennyLane? The version is not going to be overwritten during the PL installation right? (unless the versions conflict with PL's dep requirements 🤔)

Reply:
There shouldn't be any issue as PL only checks if the installed pennylane-lightning version is satisfying the minimum (v0.28.0) if not it'll installs the stable release.